journal-of-porphyrins-and-phthalocyanines.csl

```diff
@@ -0,0 +1,251 @@
+    <author>
+      <name>Jeremy Kuhne</name>
+      <email>jakuhne@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="chemistry"/>
+    <issn>1088-4246</issn>
+    <eissn>1099-1409</eissn>
+    <summary>In agreement with JPP http://jpp.fontismedia.com/jpp/guides/Notice_to_Authors.pdf</summary>
+    <published>2018-05-25T02:40:07+00:00</published>
+    <updated>2018-05-25T03:40:07+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="page-range-delimiter">-</term>
+      <term name="available at">available from</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor">
+      <name delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <group suffix=".">
+      <names variable="author">
+        <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="18" et-al-use-last="true" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+        <label form="short" prefix=", "/>
+        <substitute>
+          <names variable="editor"/>
+          <text macro="title"/>
+        </substitute>
+      </names>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if match="none" variable="issued">
+        <choose>
+          <if variable="DOI">
+            <text value="DOI:"/>
+            <text variable="DOI"/>
+          </if>
+          <else-if variable="URL">
+            <group prefix="[" suffix="] ">
+              <text term="accessed" text-case="capitalize-first" suffix=" "/>
+              <date variable="accessed">
+                <date-part name="month" suffix=" "/>
+                <date-part name="day" suffix=", "/>
+                <date-part name="year"/>
+              </date>
+            </group>
+            <group delimiter=": ">
+              <text term="available at" text-case="capitalize-first"/>
+              <text variable="URL"/>
+            </group>
+            <group delimiter=". ">
+              <choose>
+                <if type="webpage">
+                  <date variable="issued" prefix="Published " form="text"/>
+                </if>
+              </choose>
+            </group>
+          </else-if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" text-case="title"/>
+      </if>
+      <else>
+        <text value=""/>
```

Comment:
What's the goal here? If you actually don't want a title for some item types, you can just not have an `else` condition.

Reply:
Thank you for pointing these out. If you can't tell I'm new to this. I only have a little experience and that's with java. I started on this before I realized that I could ask someone else to make these. I used the visual editor so I was uncertain about just deleting functions.

Reply:
I removed this already in the style, so only thing left is the et-al quesiton.